libpod/healthcheck_linux.go

```diff
@@ -28,6 +28,7 @@ func (c *Container) createTimer(interval string, isStartup bool) error {
 	}
 
 	var cmd = []string{}
+	cmd = append(cmd, "--property", "LogLevelMax=notice")
```

Comment:
Is this a leftover?

Reply:
No, it was introduced as planned. (assuming by leftover, you meant code that need not be there)

Reply:
Thank you. If it's deliberate, would it not be better to define it in the declaration? I don't know Go best practices, but `declare empty, then append` looks jarring to my eye. It costs me energy, trying to figure out why it was done that way, and will presumably be a burden to future maintainers. But perhaps Go does things differently?

Reply:
removed

Reply:
adjusted in code, thanks!